shared/fixes/bash/auditd_data_retention_space_left.sh

```diff
@@ -2,7 +2,7 @@
 . /usr/share/scap-security-guide/remediation_functions
 populate var_auditd_space_left
 
-grep -q ^space_left /etc/audit/auditd.conf && \
+grep -q "^space_left[[:space:]]*=" /etc/audit/auditd.conf && \
   sed -i "s/^space_left[[:space:]]*=.*/space_left = $var_auditd_space_left/g" /etc/audit/auditd.conf
 if ! [ $? -eq 0 ]; then
     echo "space_left = $var_auditd_space_left" >> /etc/audit/auditd.conf
```

Comment:
The if condition can be avoided by chaining the `echo` to the previous compound expression using `&&`.

Reply:
This is a mistaken comment and you can either disregard it, or consider chaining using the `||` operator. Bash logical operators have the same precedence and are [left-associative](https://stackoverflow.com/a/930505).

Reply:
I have changed the `if` to `||` approach.